packages/agoric-cli/src/json-http-client-node.js

```diff
@@ -12,7 +12,7 @@ export const makeJsonHttpClient = ({ http }) => {
   /**
    * @param {object} request
    * @param {string} request.hostname
-   * @param {number} request.port
+   * @param {string | number} request.port
```

Comment:
I don't think we should accommodate this kind of misuse... `port` is logically a number, and is documented to be of type "number" by the underlying Node.js functionality: https://nodejs.org/docs/latest/api/http.html#httprequestoptions-callback
> <code>port</code> <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type" class="type">&lt;number&gt;</a> Port of remote server. <strong>Default:</strong> <code>defaultPort</code> if set, else <code>80</code>.

Instead, we should leave this as-is and fix any incorrect call sites.
```suggestion
   * @param {number} request.port
```